jenkins-ci/Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+pipeline {
+    agent any
+    stages {
+        stage('all-in-one: clean environment') { 
+            steps { 
+                sh 'oc get all'
+                sh 'oc delete all,template,daemonset -l jaeger-infra'
+                sh 'oc get all'
+            }
+        }
+        stage('all-in-one: deploy Jaeger') { 
+            steps { 
+                sh 'oc get all'
```

Comment:
I'm a bit concerned about the verbosity of this. While I see the value in getting the state of `oc get all` after every step, it would probably be better to add a message before, explaining what's going on.

Would you have a sample output for this build?